framework/bee-py/poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.5 and should not be changed by hand.
```

Comment:
Do we need a policy on updating of the lock file.

Sometimes libraries don't check this in, but the benefit of checking in is consistency (especially for applications)
If we do want to go with checkins we may want to ensure consistent use of tool & python versions (as dependencies can resolve differently based on python version in use). One possibility (perhaps in future) is to do as part of a build process

In general though the updates are progressive and we're in early stages so no blocking concern.

Reply:
we can remove it for know to move it more agile, and define the policy on updating of the lock file.

Reply:
I just remove it for now.